### ISSUE TITLE
Black Screen

### ISSUE THREAD

**Issue by jplima1** [RetroAchievements/RALibretro#10]
I've just downloaded latest version and it gives me black screen, only sound working.

My laptop graphics card: Intel HD Graphics 3000
Windows 10 x64
6GB RAM
Directx 12.

**Comment by jplima1** [comment]
Any of them, Meleu.

**Comment by leiradel** [comment]
@jplima1 can you attach the `log.txt` file? You can find it in the same folder as `RALibretro.exe`.

**Comment by JuliaWolska** [comment]
I've recieved a log.txt from BerserkerBR, who has similiar problem
[log.txt](https://github.com/RetroAchievements/RALibretro/files/1949440/log.txt). He also told me he is working on Windows 7, and RetroArch is crashing for him on launch. I guess it's an issue related directly to Libretro engine, not specifically to RALibretro

**Comment by leiradel** [comment]
@JuliaWolska this log says the user has OpenGL 2.1, and there's an OpenGL error further down the log. Please ask him/her to update the drivers and see if the issue goes away.

**Comment by JuliaWolska** [comment]
Yeah I already talked about it with him. Does Libretro require Open GL newer than 2.1? It seems that his gpu (intel 45 express) doesn't support newer...

**Comment by leiradel** [comment]
My VM also reports OpenGL 2.1, and it does support the functionalities RALibretro uses. Maybe the driver has a bug and an update solves it.
